src/Php84/Resources/stubs/RoundingMode.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+if (\PHP_VERSION_ID < 80400 && extension_loaded('bcmath')) {
```

Comment:
RoundingMode is not part of BCMath (BCMath reuses the enum introduced for `round`)

Reply:
Removed that. Good catch!